src/platform/silabs/efr32/ThreadStackManagerImpl_CMSIS.hpp

```diff
@@ -0,0 +1,138 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *    Copyright (c) 2019 Nest Labs, Inc.
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ *    @file
+ *          Contains non-inline method definitions for the
+ *          GenericThreadStackManagerImpl_FreeRTOS<> template.
+ */
+
+#include <lib/support/CodeUtils.h>
+#include <lib/support/logging/CHIPLogging.h>
+#include "../ThreadStackManagerImpl_CMSIS.h"
+#include <platform/OpenThread/OpenThreadUtils.h>
+#include <platform/ThreadStackManager.h>
+#include <platform/internal/CHIPDeviceLayerInternal.h>
+
+namespace chip {
+namespace DeviceLayer {
+namespace Internal {
+
+template <class ImplClass>
+CHIP_ERROR ThreadStackManagerImpl_CMSIS<ImplClass>::DoInit(void)
+{
+    CHIP_ERROR err = CHIP_NO_ERROR;
+    return err;
+}
+
+template <class ImplClass>
+CHIP_ERROR ThreadStackManagerImpl_CMSIS<ImplClass>::_StartThreadTask(void)
+{
+    return CHIP_NO_ERROR;
+}
+
+template <class ImplClass>
+void ThreadStackManagerImpl_CMSIS<ImplClass>::_LockThreadStack(void)
+{
+}
+
+template <class ImplClass>
+bool ThreadStackManagerImpl_CMSIS<ImplClass>::_TryLockThreadStack(void)
+{
+    return true;
+}
+
+template <class ImplClass>
+void ThreadStackManagerImpl_CMSIS<ImplClass>::_UnlockThreadStack(void)
+{
+}
+
+#if CHIP_DEVICE_CONFIG_ENABLE_THREAD_SRP_CLIENT
+template <class ImplClass>
+void ThreadStackManagerImpl_CMSIS<ImplClass>::_WaitOnSrpClearAllComplete()
+{
+    // Only 1 task can be blocked on a srpClearAll request
```

Comment:
Do we have to keep all the commented code in this file

Reply:
Actually, we no not need this file... must have slipped through